src/js/data/shop/cart.js

```diff
@@ -0,0 +1,63 @@
+import { CrudConsumer, CrudConsumerObject } from "consumerjs";
+
+import { API_ROOT } from "../../constants";
+
+class Cart extends CrudConsumerObject {}
+
+export class CartConsumer extends CrudConsumer {
+    constructor(endpoint = `${API_ROOT}api/v1/shop/cart`, objectClass = Cart) {
+        super(endpoint, objectClass);
+    }
+
+    fetch() {
+        return this.get("/");
+    }
+
+    rotate(id, direction) {
```

Comment:
not sure what a rotating cart is supposed to be ;)

Reply:
Oops :3